extensions/positron-python/python_files/positron/positron_ipykernel/tests/test_inspectors.py

```diff
@@ -917,7 +913,10 @@ def test_get_child(value: Any, key: Any, expected: Any) -> None:
 
 @pytest.mark.skipif(sys.version_info < (3, 10), reason="requires Python 3.10 or higher")
 def test_inspect_ibis_exprs() -> None:
-    import ibis
+    try:
+        import ibis  # type: ignore [reportMissingImports] for 3.13
+    except ImportError:
+        return  # skip this test if ibis is not installed
```

Comment:
We could instead mark this as `xfail` ("expected to fail", [docs](https://docs.pytest.org/en/stable/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail)) when the Python version is >= 3.13.

If we also enable the `strict` parameter ([docs](https://docs.pytest.org/en/stable/how-to/skipping.html#strict-parameter)), CI will fail if the test unexpectedly passed i.e. if ibis was somehow installed on >= 3.13. This might be a neat way to make sure we clean this up when we update `pinned-test-requirements.txt` once ibis is supported.

Reply:
Yeah I like this strategy!

Reply:
I like this plan too! updated 👍